src/cloud/project/new-relic.md

```diff
@@ -268,55 +274,71 @@ Prerequisites
 
-   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-alert-policies.png){:width="650px"}
+      {:.bs-callout-info}
+      If the policy is missing, submit a Magento support ticket. Include your project ID in the ticket. The policy is available on Pro and Starter Production environments only.
 
-   {:.bs-callout-info}
-   If you do not see these alert policies, submit a Magento support ticket. Include your project ID in the ticket. These policies are available on Pro and Starter Production environments only.
+1. Click **Managed Alerts for Magento Commerce** to review the alert conditions defined in the policy.
 
-1. Click an alert policy name to review the threshold conditions that trigger alerts.
+   ![Managed alerts list]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-magento-alert-conditions.png){:width="650px"}
 
 ### Configure notification channels
 
-To use the Adobe-generated alert policies to monitor your Production sites, you must configure notification channels and map them to alert policies. Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
+To use the Managed Alerts for Magento Commerce policy to monitor your Production sites, you must configure at least one notification channel and map it to the alert policy.
 
-New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.
+Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
 
-The following instructions describe the high level steps to configure a notification channel to receive alert messages triggered by an alert policy.
+New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.
 
 {:.procedure}
 To configure a notification channel:
 
 1. Choose a notification channel type, and complete any [prerequisite steps] required to connect the channel with the New Relic service.
 
-1. [Log in to your New Relic account][New Relic login].
+1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].
+
+1. Create a new notification channel.
+
+   -  In the top navigation menu, click **Alerts & AI** to open the _Applied Intelligence_ page.
+
+   -  In the left navigation, click **Notification channels**.
+
```

Comment:
Is it important to comment about selecting an account or project from the drop-down list here like previously?

Reply:
I didn't think so, which is why I left it out.